app/views/unattended/report_templates/subscription_-_entitlement_report.erb

```diff
@@ -22,35 +22,39 @@ require:
 <%-   search: should_filter && "pools_expiring_in_days = #{days_from_now}" -%>
 <%- ).each_record do |host| -%>
 <%-   host.pools.expiring_in_days(days_from_now).sort_by { |p| p.end_date }.each do |pool| -%>
-<%-     report_row(
-          'Host Name': host.name,
-          'Organization': host.organization,
-          'Lifecycle Environment': host.single_lifecycle_environment.name,
-          'Content View': host.single_content_view.name,
-          'Host Collections': host_collections_names(host),
-          'Virtual': host.virtual,
-          'Guest of Host': host.hypervisor_host,
-          'OS': host.operatingsystem,
-          'Arch': host.architecture,
-          'Sockets': host.sockets,
-          'RAM': host.ram,
-          'Cores': host.cores,
-          'SLA': host_sla(host),
-          'Products': host_products_names(host),
-          'Last Checkin': last_checkin(host).to_s,
-          'Subscription Name': sub_name(pool),
-          'Subscription Type': pool.type,
-          'Subscription Total Quantity': pool.quantity,
-          'Subscription Total Consumed': pool.consumed,
-          'Subscriptions Consumed by Host': host.filtered_entitlement_quantity_consumed(pool),
-          'Subscription SKU': sub_sku(pool),
-          'Subscription Contract': pool.contract_number,
-          'Subscription Account': pool.account_number,
-          'Subscription Start': pool.start_date,
-          'Subscription End': pool.end_date,
-          'Subscription Guest': registered_through(host),
-          'Days Remaining': pool.days_until_expiration
-          ) -%>
+<%-     unless host.organization.simple_content_access? -%>
```

Comment:
If a host's organization is in SCA, the host probably won't have any `pools` attached. So checking `simple_content_access?` here is kind of useless. I'd say just leave it as-is here, maybe, and let the other report cover SCA orgs.

Reply:
Oh I thought of one reason to filter it -- if a host used to be in an SCA org, the old pool information will still be there. So, by filtering out the simple content access or hosts, users won't see old information